tfx/extensions/experimental/kfp_compatibility/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Lint as: python2, python3
```

Comment:
nit: if you are about to push new changes, can you please add __init__py in the following places?

- tfx/extensions/experimental/__init__.py
- tfx/extensions/experimental/kfp_compatibility/proto/__init__.py

Without these files, pip packaging omits the files in these folders. 
If the PR is mergeable, you can leave it as-is, and I can address this issue afterwards.

Reply:
Thanks for letting me know, I've add the __init__.py files to the two locations.